mail_notification_with_history/data/mail_data.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+  <template
+        id="mail_notification_with_history.mail_notification_layout"
+        inherit_id="mail.mail_notification_layout"
+    >
+    <xpath expr="//div[@t-out='signature']" position="after">
+
+      <t
+                t-set="message_history"
+                t-value="message._get_notification_message_history()"
+            />
+      <t t-if="message_history">
+        <p class="message_history_title">Discussion history in Odoo:</p>
+        <ul>
+          <t t-foreach="message_history" t-as="msg">
+            <li class="message_history_item">
+            <p><t t-esc="msg.author_id.name or 'unknown'" /> commented on <span
```

Comment:
`t-esc` is soft deprecated, could we switch to `t-out` here?

Reply:
to be addressed